recipe/meta.yaml

```
@@ -13,18 +12,19 @@ source:
 build:
   entry_points:
     - instructor = instructor.cli.cli:app
-  noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<310]
   number: 0
-
+# Though it is available upstream for 3.9, we’ll limit the package to Python 3.10 and above,
+# since the newer typing syntax breaks all 3.9 builds.  
 requirements:
   host:
-    - python {{ python_min }}
+    - python
     - poetry-core
     - hatchling
     - pip
   run:
-    - python >={{ python_min }}
+    - python
     - openai >=1.52.0,<2.0.0
     - pydantic >=2.8.0,<3.0.0
     - docstring_parser >=0.16.0,<0.17.0
```

```suggestion
    - docstring_parser >=0.16.0,<1.0
```

https://github.com/instructor-ai/instructor/blob/1.7.9/pyproject.toml#L8

Some other dependencies that need updated:

`- requests >=2.32.3,<3.0.0`

`- jiter >=0.6.1,<0.9`

`- jinja2 >=3.1.4,<4.0.0`

---

Adding some additional deps. Let me know if these work!